src/notifications.ts

```diff
@@ -11,11 +21,39 @@ export enum NotificationStatus {
 }
 
 export enum Actions {
-  UpgradeProjectToPro = 'project.upgrade-pro',
-  UpgradeProjectToPayg = 'project.upgrade-payg',
+  UpgradeProjectToPro = 'project.upgrade',
   SchedulePostgresRestart = 'postgresql.restart',
 }
 
+export interface PostgresRestartRequired {
```

Comment:
this is no longer being used right?

Reply:
Ah yeah, needs to be subsumed by Actions. Will update